context/context.go

```diff
@@ -63,7 +63,7 @@ func doInit() {
 
 	utils.InitDir(config.LocalStateDir)
 
-	initMyUUID()
+	MyUUID = initMyUUID()
```

Comment:
where are you using this MyUUID ?. if you are not using then you can ignore return value of initMyUUID()